circus-train-integration-tests/pom.xml

```diff
@@ -172,6 +165,28 @@
       <artifactId>protobuf-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I'd be wary of setting these to version numbers other than what the Hive/Hadoop dependencies are using. I guess this works but I just know how fragile things can get as the newer versions aren't backwards compatible and the only reason we're not seeing an issue now is we haven't hit certain code paths that would trigger problems.